src/vendor/libc/ci/README.md

```diff
@@ -74,8 +74,7 @@ The remaining architectures look like:
 ## QEMU
 
 Lots of the architectures tested here use QEMU in the tests, so it's worth going
-over all the crazy capabilities QEMU has and the various flavors in which we use
-it!
+over all the capabilities QEMU has and the various flavors in which we use it!
```

Comment:
this change can't work; this is a vendored copy of the https://github.com/rust-lang/libc crate, and would need to be fixed there, that's why CI is failing.